.travis.yml

```diff
@@ -0,0 +1,16 @@
+dist: trusty
+install:
+        - sudo apt update
+
+        # Packages for Rosette-VM building
+        - sudo apt install g++ g++-multilib
```

Comment:
I think that we should move the bottom level dependencies to concrete subprojects builds, so we keep .travis.yml as clear as possible and leave here only Travis related settings.
So if it possible I suggest moving this "apt-get install" stuff to concrete build.sh files.

Reply:
I'm not sure what you mean. There is no `rosette/build.sh`.

Reply:
I think I see now... I'm trying matrix builds, too.